lib/mongery.rb

```diff
@@ -28,6 +28,10 @@ def find(*args)
       build_query.where(*args)
     end
 
+    def count
```

Comment:
What do you think about adding `project` method in the Query class so that you can somewhat call arbitrary columns/function there? Need to make sure it's not vulnerable to SQL injection.